sdk/keyvault/keyvault-secrets/review/keyvault-secrets.api.md

```diff
@@ -113,12 +101,11 @@ export interface Secret {
 
 // @public
 export class SecretClient {
-    constructor(endPoint: string, credential: TokenCredential, pipelineOrOptions?: ServiceClientOptions | NewPipelineOptions);
+    constructor(endPoint: string, credential: TokenCredential, pipelineOptions?: PipelineOptions);
     backupSecret(secretName: string, options?: RequestOptionsBase): Promise<Uint8Array | undefined>;
     beginDeleteSecret(name: string, options?: SecretPollerOptions): Promise<PollerLike<PollOperationState<DeletedSecret>, DeletedSecret>>;
     beginRecoverDeletedSecret(name: string, options?: SecretPollerOptions): Promise<PollerLike<PollOperationState<SecretProperties>, SecretProperties>>;
     protected readonly credential: TokenCredential;
-    static getDefaultPipeline(credential: TokenCredential, pipelineOptions?: NewPipelineOptions): ServiceClientOptions;
```

Comment:
Now that there is no more `getDefaultPipeline`,  what is the replacement? How can a user get the default pipeline, tweak it and return it back?

Reply:
I added `PipelineOptions.requestPolicyFilter` for that, it gets called at the end of `createPipelineFromOptions` with the list of policy factories that got created.  Any thoughts on the name?  Could call it `requestPolicyFactoryFilter` but that's kinda long.